config/application.rb

```diff
@@ -105,9 +105,6 @@ class Application < Rails::Application
     # Use Resque to run ActiveJobs (including sending delayed mail):
     config.active_job.queue_adapter = :resque
 
-    # TODO: Remove with Rails 8.0 where this option will be deprecated
-    config.active_job.enqueue_after_transaction_commit = :always
```

Comment:
I've set  `enqueue_after_transaction_commit = true` on our job classes since the option is deprecated. An alternative would be to set `ActiveJob::Base.enqueue_after_transaction_commit = true` in an initializer.

As an additional note, [it might be enabled by default](https://redirect.github.com/rails/rails/pull/53375#issuecomment-2555694252) in the future.